engine/apps/chatops_proxy/utils.py

```diff
@@ -48,20 +48,18 @@ def get_slack_oauth_response_from_chatops_proxy(stack_id) -> dict:
     return slack_installation.oauth_response
 
 
-def register_oncall_tenant(service_tenant_id: str, cluster_slug: str, stack_id: int):
+def register_oncall_tenant_with_async_fallback(org):
     """
     register_oncall_tenant tries to register oncall tenant synchronously and fall back to task in case of any exceptions
     to make sure that tenant is registered.
     First attempt is synchronous to register tenant ASAP to not miss any chatops requests.
     """
-    client = ChatopsProxyAPIClient(settings.ONCALL_GATEWAY_URL, settings.ONCALL_GATEWAY_API_TOKEN)
+    service_tenant_id = str(org.uuid)
+    cluster_slug = org.stack_slug
+    stack_id = org.stack_id
+
     try:
-        client.register_tenant(
-            service_tenant_id,
-            cluster_slug,
-            SERVICE_TYPE_ONCALL,
-            stack_id,
-        )
+        register_oncall_tenant(org)
     except Exception as e:
```

Comment:
down below in the call to `register_oncall_tenant_async.apply_async`, aren't you missing the new `stack_slug` kwarg to pass in?

Reply:
Yep, nice catch